README.md

```diff
@@ -430,8 +430,8 @@ A theme directory contains at least 2 files:
 These files need to be made executable. Many theme directories also contain:
 
 - `theme.ron`: which contains additional configuration options specific to the theme.
-- `polybar.ini`: a configuration file for the `polybar` application. *Need to have polybar installed!*
-- `picom.conf`: a configuration file for the `picom` compositor. *Need to have picom installed!*
+- `polybar.ini`: a configuration file for the `polybar` application. _Need to have polybar installed!_
+- `picom.conf`: a configuration file for the `picom` compositor. _Need to have picom installed!_
```

Comment:
As you're already on it, how about
```suggestion
- `polybar.ini`: a configuration file for the `polybar` application. _You need to have polybar installed!_
- `picom.conf`: a configuration file for the `picom` compositor. _You need to have picom installed!_
```

Reply:
Thanks, and good catch as always @Eskaan! :)